gslib/utils/text_util.py

```diff
@@ -167,8 +167,6 @@ def EncodeStringAsLong(string_to_convert):
     Long that represents the input string.
   """
   hex_bytestr = codecs.encode(six.ensure_binary(string_to_convert), 'hex_codec')
-  if six.PY3:
```

Comment:
Was removing these two lines intentional? Since Python 3 doesn't have long

Reply:
Yeah; it's already defined higher up in the module.  Also, for some reason, having it here resulted in an error in PY2 about trying to use the variable "long" before it was defined (apparently Python realizes it was *potentially* assigned in an if-block above and treats it as an undefined local variable or something?).

Reply:
Got it! Merging :)